catalogue/webapp/pages/works.js

```diff
@@ -9,16 +8,10 @@ const Works = () => (
     title='Image catalogue search | Wellcome Collection'
     description='Search through the Wellcome Collection image catalogue'
   >
-    <div className={grid({
-      s: 12,
-      m: 12,
-      l: 12,
-      xl: 12
-    })}>
-      <style dangerouslySetInnerHTML={{ __html: criticalCss }} />
-      <PageDescription title='Search our images' modifiers={{hidden: true}} />
-      <InfoBanner text={`Coming from Wellcome Images? All freely available images have now been moved to the Wellcome Collection website. Here we're working to improve data quality, search relevance and tools to help you use these images more easily`} cookieName='WC_wellcomeImagesRedirect' />
-    </div>
+    <style dangerouslySetInnerHTML={{ __html: criticalCss }} />
```

Comment:
I ran in to a snag injecting our CSS like this in Cardigan – `content: ''` in pseudo elements was being output as `content: &apos;&apos;` (or similar). Is this the same here?

I ended up linking to the file, but that's obviously not an option here.

Reply:
Nope. What were you using to inject the css?

Reply:
A style tag with `dangerouslySetInnerHTML`. Don’t think there was anything else different. But if it’s working, then no worries.